DQMOffline/EGamma/plugins/PhotonDataCertification.cc

```diff
@@ -85,17 +85,18 @@ float PhotonDataCertification::invMassZtest(string path, TString name,
   if (TestElem == 0) return 0;
   TH1F *TestHist = TestElem->getTH1F();
   if (TestHist == 0) return 0;
+  RooMsgService::instance().setGlobalKillBelow(RooFit::WARNING) ;
```

Comment:
indeed - looking further, my call is not adding significantly to the # of calls to RooMsgService::instance() - there are already ~10 of them (one per printout line that I'm trying to quiet)